src/agent_factory/generation.py

```diff
@@ -1,70 +1,27 @@
 import json
-import shutil
 import uuid
 from datetime import datetime
 from pathlib import Path
 
 import dotenv
 import fire
 from any_agent import AgentConfig, AgentFramework, AnyAgent
-from any_agent.config import MCPStdio
 from any_agent.tools import search_tavily, visit_webpage
 from pydantic import BaseModel, Field
 
 from agent_factory.instructions import INSTRUCTIONS
 from agent_factory.prompt import UserPrompt
-from agent_factory.tools import search_mcp_servers
+from agent_factory.tools import read_file, search_mcp_servers
 
 dotenv.load_dotenv()
 
-repo_root = Path.cwd()
```

Comment:
I went with asking forgiveness rather than permission here.

@khaledosman I assume the UI was somehow relying on an structure.

I would like to understand how these folders were used in the UI.

It would be great if we could all use the same generations (`tests/assets`) in the UI as we will use in the "backend" to guide iterations.